test/suites/integration/ami_test.go

```diff
@@ -61,12 +61,7 @@ var _ = Describe("AMI", func() {
 	})
 	It("should use the most recent AMI when discovering multiple", func() {
 		// choose an old static image
-		parameter, err := env.SSMAPI.GetParameter(&ssm.GetParameterInput{
-			Name: aws.String("/aws/service/eks/optimized-ami/1.23/amazon-linux-2/amazon-eks-node-1.23-v20230322/image_id"),
-		})
-		Expect(err).To(BeNil())
-		oldCustomAMI := *parameter.Parameter.Value
-		nodeClass.Spec.AMIFamily = &v1beta1.AMIFamilyCustom
```

Comment:
Do you know why we were setting this to custom? Does it make sense to remove in this case?

Reply:
Doesn't appear to be any particular reason, we don't currently surface enough information to the E2E test to generate a `NodeConfig` object for AL2023's user data. Since the AMI family doesn't effect our AMI selector logic this change is safe to make.